src/PFire.Core/Protocol/Messages/Outbound/GameServerSendAll.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using PFire.Core.Session;
+
+namespace PFire.Core.Protocol.Messages.Outbound
+{
+    internal sealed class GameServerSendAll : XFireMessage
+    {
+        public GameServerSendAll(int gameId) : base(XFireMessageType.GameServerSendAll)
```

Comment:
Should these be unsigned ints? Or are you planning to rebase this with the usigned ints fix and fix it there after?

Reply:
I figure to fix it after, base this on the int base and then fix it when the time comes.